galaxy_ng/tests/integration/api/test_community.py

```diff
@@ -524,3 +524,62 @@ def get_roles(page_size=1, order_by='created'):
 
     # cleanup
     clean_all_roles(ansible_config)
+
+
+@pytest.mark.community_only
+def test_v1_role_tag_filter(ansible_config):
+    """" Tests if v1 roles are auto-sorted by created """
+
+    config = ansible_config("admin")
```

Comment:
if just testing search, can probably test with a non admin profile

Reply:
Sync requires an admin to run.